packages/grant-explorer/src/features/round/ViewProjectDetails.tsx

```diff
@@ -33,6 +33,26 @@ import RoundEndedBanner from "../common/RoundEndedBanner";
 import PassportBanner from "../common/PassportBanner";
 import markdown from "../../app/markdown";
 
+const CalendarIcon = (props: React.SVGProps<SVGSVGElement>) => {
```

Comment:
is this icon not available in heroicons? if we need it maybe we can split it into it's own file

Reply:
Agreed - this is available in heroicons: https://github.com/tailwindlabs/heroicons/blob/master/src/24/solid/calendar.svg

Reply:
Unfortunately it's not the same icon. I tried this one originally, but it's not filled in like the one in Figma. Good idea on the split though.

Reply:
let's wait until we decided how we do icons in general for the splitting and ship this first

Reply:
@hmrtn